libs/repositories/src/UsdRepository.ts

```diff
@@ -0,0 +1,16 @@
+import { get } from 'http';
+
+export interface UsdRepository {
+  getPriceDayUsd(tokenAddress: string, date: Date): Promise<number>;
```

Comment:
main signature I'd like to suggest

Reply:
It sounds weird. What about `getDailyUsdPrice`